.pylintrc

```diff
@@ -65,7 +65,7 @@ confidence=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=locally-disabled,duplicate-code,too-many-arguments,too-many-instance-attributes,invalid-name,too-few-public-methods,no-name-in-module,import-error,missing-docstring,map-builtin-not-iterating,buffer-builtin,long-suffix,import-star-module-level,xrange-builtin,apply-builtin,input-builtin,raising-string,reload-builtin,hex-method,delslice-method,execfile-builtin,unpacking-in-except,round-builtin,backtick,print-statement,using-cmp-argument,oct-method,raw_input-builtin,long-builtin,unichr-builtin,filter-builtin-not-iterating,metaclass-assignment,old-raise-syntax,cmp-builtin,old-octal-literal,next-method-called,standarderror-builtin,setslice-method,basestring-builtin,suppressed-message,range-builtin-not-iterating,indexing-exception,dict-view-method,getslice-method,cmp-method,coerce-builtin,old-division,coerce-method,old-ne-operator,file-builtin,intern-builtin,parameter-unpacking,reduce-builtin,dict-iter-method,zip-builtin-not-iterating,useless-suppression,unicode-builtin,nonzero-method,no-absolute-import
+disable=inconsistent-return-statements,locally-disabled,duplicate-code,too-many-arguments,too-many-instance-attributes,invalid-name,too-few-public-methods,no-name-in-module,import-error,missing-docstring,map-builtin-not-iterating,buffer-builtin,long-suffix,import-star-module-level,xrange-builtin,apply-builtin,input-builtin,raising-string,reload-builtin,hex-method,delslice-method,execfile-builtin,unpacking-in-except,round-builtin,backtick,print-statement,using-cmp-argument,oct-method,raw_input-builtin,long-builtin,unichr-builtin,filter-builtin-not-iterating,metaclass-assignment,old-raise-syntax,cmp-builtin,old-octal-literal,next-method-called,standarderror-builtin,setslice-method,basestring-builtin,suppressed-message,range-builtin-not-iterating,indexing-exception,dict-view-method,getslice-method,cmp-method,coerce-builtin,old-division,coerce-method,old-ne-operator,file-builtin,intern-builtin,parameter-unpacking,reduce-builtin,dict-iter-method,zip-builtin-not-iterating,useless-suppression,unicode-builtin,nonzero-method,no-absolute-import
```

Comment:
We should try and whittle the number of disabled rules down as this is starting to get excessive. Not as part of this PR though.